Assets/MRTK/Core/Utilities/Gltf/Serialization/GltfUtility.cs

```diff
@@ -10,7 +10,7 @@
 using System.Threading.Tasks;
 using UnityEngine;
 
-#if WINDOWS_UWP
+#if ENABLE_WINMD_SUPPORT
```

Comment:
Should these be changed repo-wide? Why are they just targeted in the glTF files?

Reply:
Those were the files where I saw a difference with these, but I think it can be undone with no downsides

Reply:
I'm good leaving this! I think it's technically more correct